src/main/resources/templates/db_scripts/data-migration/13.8.0/populate_search_locations_ids.sql

```diff
@@ -0,0 +1,29 @@
+-- Update searchLocationIds arrays co contains all location ids and all location holding permanent location ids
```

Comment:
@azizbekxm do you remember what issue we had that triggered us to add this declaration
<#if mode.name() == "UPDATE">
https://github.com/folio-org/mod-invoice-storage/pull/176/files#diff-974794201fff65e2fd3e90a32f2c532fac75043094026e981a0b0ec7576143f8R1

Reply:
we can join, but db in unit tests are different. check pr, you should the same to pass tests.

Reply:
referring table is not exists in unit test pgadmin db, so need so fix